glide.yaml

```diff
@@ -1,39 +1,56 @@
 package: github.com/bblfsh/server
 import:
+- package: github.com/Sirupsen/logrus
+  version: ^1.0.2
 - package: github.com/bblfsh/sdk
+  version: 9db6b2ee9500857b54f8d40165dc92aba482aadf
```

Comment:
it could be, but master depends on the cache, it could use a commit previous to this one if `glide` cached it when master was at this point.

In this way you are sure to use this specific commit and if you have to run `glide up` you wouldn't have problems with this cache/version issues